### PR TITLE
Update request api with ODLM new design

### PR DESCRIPTION
**Pull request by DanielXLee**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

### CODE REVIEW

File: deploy/crds/operator.ibm.com_v1alpha1_operandrequest_cr.yaml [hunk 1]
```diff
@@ -4,36 +4,20 @@ metadata:
...
-  - name: ibm-helm-repo-operator
-    state: absent
-  - name: ibm-management-repo-operator
-    state: absent
+  requests:
+  - operand: ibm-cert-manager-operator
+  - operand: ibm-mongodb-operator
+  - operand: ibm-iam-operator
+  - operand: ibm-monitoring-exporters-operator
+  - operand: ibm-healthcheck-operator
```
Comment (gyliu513) [inline deploy/crds/operator.ibm.com_v1alpha1_operandrequest_cr.yaml]: so we only have five on board operators?

**Comment by gyliu513** [comment]
Same question, can you make this as two commits?

1) Manual code change
2) Code generated by `make code-dev`

File: deploy/crds/operator.ibm.com_v1alpha1_operandrequest_cr.yaml (hunk 1 above)
Comment (gyliu513) [reply deploy/crds/operator.ibm.com_v1alpha1_operandrequest_cr.yaml]: Ah, I see, other services are still `absent`

**Comment by DanielXLee** [comment]
@gyliu513 got it, next time, I'll separate as 2 commits,  genera, following code is auto-generated, 
```
├── pkg
│   ├── apis
│   │   └── operator
│   │       └── v1alpha1
│   │           └── zz_generated.deepcopy.go

deploy
├── crds
│   ├── operator.ibm.com_operandconfigs_crd.yaml
│   ├── operator.ibm.com_operandregistries_crd.yaml
│   ├── operator.ibm.com_operandrequests_crd.yaml
│   ├── operator.ibm.com_v1alpha1_operandconfig_cr.yaml
│   ├── operator.ibm.com_v1alpha1_operandregistry_cr.yaml
│   └── operator.ibm.com_v1alpha1_operandrequest_cr.yaml
├── olm-catalog
│   └── operand-deployment-lifecycle-manager
│       ├── 0.0.1
│       │   ├── operand-deployment-lifecycle-manager.v0.0.1.clusterserviceversion.yaml
│       │   ├── operator.ibm.com_operandconfigs_crd.yaml
│       │   ├── operator.ibm.com_operandregistries_crd.yaml
│       │   └── operator.ibm.com_operandrequests_crd.yaml
│       └── operand-deployment-lifecycle-manager.package.yaml

```

**Pull request merged**